app/services/return-requirements/setup.service.js

```diff
@@ -20,8 +21,9 @@ const SessionModel = require('../../models/session.model.js')
  */
 async function go (sessionId) {
```

Comment:
I know you didn't write the JSDoc comment for this file, but can you check if line 14 is correct? It mentions the 'select reason page' and I'm not sure if that is meant to say 'set up page' instead.